.github/workflows/mmctl-test-template.yml

```diff
@@ -0,0 +1,50 @@
+name: mmctl CI
+on:
+  workflow_call:
+    inputs:
+      datasource:
+        required: true
+        type: string
+      drivername:
+        required: true
+        type: string
+env:
+  go-version: "1.19.5"
+jobs:
+  run-mmctl-tests:
+    runs-on: ubuntu-latest-8-cores
+    env:
+      COMPOSE_PROJECT_NAME: ghactions
+      BUILD_IMAGE: mattermost/mattermost-build-server:20230118_golang-1.19.5
+    steps:
+      - name: Checkout mattermost-server
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+      - name: Setup Go
+        uses: actions/setup-go@4d34df0c2316fe8122ab82dc22947d607c0c91f9 # v4.0.0
+        with:
+          go-version: ${{ env.go-version }}
```

Comment:
Is there a reason for hardcoding the go version? Setup-go can autodetect this using go.mod file. 

https://github.com/actions/setup-go#getting-go-version-from-the-gomod-file

Reply:
Yes, we need to pin it to the exact patch release version.